pkg/manifest/manifest.go

```diff
@@ -895,11 +896,11 @@ func (m *Manifest) SetInvocationImageAndReference(ref string) error {
 		m.Reference = bundleRef.String()
 	}
 
-	imageName, err := cnab.ParseOCIReference(bundleRef.Repository() + "-installer")
-	if err != err {
-		return errors.Wrapf(err, "could not set invocation image to %q", bundleRef.Repository()+"-installer")
+	imageName, err := cnab.ParseOCIReference(bundleRef.Repository())
+	if err != nil {
+		return errors.Wrapf(err, "could not set invocation image to %q", bundleRef.Repository())
 	}
-	imageRef, err := imageName.WithTag(dockerTag)
+	imageRef, err := imageName.WithTag(common.RandomString(bundleRef.Named.String()+dockerTag, 10))
```

Comment:
I suggest using `bundleRef.Named.WithTag(dockerTag)` instead of concatenating. Then the input string to your hash function will be a real docker reference, e.g. "example/mybundle:mytag" instead of "example/mybundlemytag".

Which seed string we use doesn't impact the solution much but when we are maintaining this code in the future, I think it would help to use that as a seed instead of something that _looks_ like a reference but isn't.

Reply:
It looks like the hash is only including the repository portion of the image reference, "example/myrepo", instead of the full reference "example/myrepo:v1.0.0". Your PR originally was including the dockerTag variable in the hash, and I was just suggesting that you alter how you concatenate the repository with the tag so that it is "example/myrepo:v1.0.0" instead of (example/myrepov1.0.0). 

Is there a reason it was dropped or can we switch both invocation images and referenced images over to generating the hash using the full image reference?